core/common/src/main/java/alluxio/network/connection/ThriftClientPool.java

```diff
@@ -60,6 +63,11 @@
   private final InetSocketAddress mAddress;
   private final long mGcThresholdMs;
 
+  private static final int THRIFT_CLIENT_POOL_GC_THREADPOOL_SIZE = 5;
```

Comment:
because netty server can support many more connections  than our current thrift server. And netty pool's max capacity is 1024 which is much larger than that of thrift client's (128).

I don't think we will ever need this many threads since the gc runs every 5 mins  now. 

I can change this if you feel this is confusing.